src/sql/src/plan/explain.rs

```diff
@@ -299,14 +308,19 @@ impl<'a> Explanation<'a> {
                 writeln!(f)?;
             }
             Get { id, .. } => match id {
-                Id::Local(local_id) => writeln!(
-                    f,
-                    "| Get %{} ({})",
-                    self.local_id_chains
-                        .get(local_id)
-                        .map_or_else(|| "?".to_owned(), |i| i.to_string()),
-                    local_id,
-                )?,
+                Id::Local(local_id) => {
+                    let get_info = self.local_id_chains.get(local_id);
+                    writeln!(
+                        f,
+                        "| Get %{} ({} ({}))",
```

Comment:
After looking at the explain plans, I'd prefer the format to be `| Get (a (l0)) (%1)`. I think the CTE name should come first because it's probably more informative for a user than the position of the plan for the CTE.

Reply:
What about `| Get t (l0) (%0)`? ie. without the parenthesis around the CTE name and the local ID/label.

Reply:
Either way works.